actionpack/lib/action_dispatch/middleware/host_authorization.rb

```diff
@@ -4,7 +4,13 @@
 
 module ActionDispatch
   # This middleware guards from DNS rebinding attacks by explicitly permitting
-  # the hosts a request can be sent to.
+  # the hosts a request can be sent to. Options are passed to this middleware
+  # from +config.host_authorization+ and can be used to configure the response
+  # application and an exclude callable.
```

Comment:
I still don't love "callable" here, that term isn't widely used in the Rails documentation.

Given that the options are both documented immediately afterwards, maybe we don't need to list them:

```suggestion
  # the hosts a request can be sent to, and is passed the options set in
  # +config.host_authorization+.
```